tests/baselines/reference/thisTypeInClasses.errors.txt

```diff
@@ -1,57 +0,0 @@
-tests/cases/conformance/types/thisType/thisTypeInClasses.ts(4,20): error TS2526: A 'this' type is available only in a non-static member of a class or interface.
-
-
-==== tests/cases/conformance/types/thisType/thisTypeInClasses.ts (1 errors) ====
-    class C1 {
-        x: this;
-        f(x: this): this { return undefined; }
-        constructor(x: this) { }
-                       ~~~~
-!!! error TS2526: A 'this' type is available only in a non-static member of a class or interface.

```

Comment:
Was this error unintended? It looks like the test meant to assert that 'this' is not accessible in statics. Why remove it with this change? I don't see any generics here.